templates/includes/delete_confirm.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}

```

Comment:
Consider adding a message or context to inform the user about what they are confirming to delete. This can be done by including a descriptive block or variable that specifies the item being deleted.